network/netcore/Cargo.toml

```diff
@@ -13,7 +13,6 @@ edition = "2018"
 async-trait = "0.1"
 bytes = "0.5.4"
 futures = { version = "0.3.4", features = ["io-compat", "compat"] }
```

Comment:
Can we drop the io-compat and compat features here while we're at it? (since we no longer depend on futures 01)